iOS.md

```diff
@@ -15,6 +15,36 @@ Refactor old sins to best of your current knowledge
   * If it isn't: Write comments for exceptional and complex code
 1. Readability trumps coolness
 
+## Spacing and Indentation
+
+* Indent using 4 spaces rather than tabs. This should be configured on the project.
+
+  ![Xcode indent settings](https://raw.githubusercontent.com/nrkno/best-practice/master/assets/xcode spacing.png)
+
+* Method braces and other braces (`if`/`else`/`switch`/`while` etc.) always open on the same line as the statement but close on a new line.
+* Don't use parentheses.
+* Tip: You can re-indent by selecting some code (or ⌘A to select all) and then Control-I (or Editor\Structure\Re-Indent in the menu).
+
+**Preferred:**
+```swift
+if video.isPlaying {
+    // Do something
+} else {
```

Comment:
I agree that sometimes it is more readable. I was often a fan of doing something like this when commenting if / else

```
if video.isPlaying {
    // Do something
} 
// comment explainig the reasoning of complex logic
else if video.isInSomeComplexLogicalState {
```

but to me consistency trumps these and consistent use of if else improved the overall readability

> Som if/else statements become more readable with else starting below the closing brace

Do you have any real world exampels where a new line would improve readability?